src/rust/madara/sources/nexoscans/res/source.json

```diff
@@ -1,10 +1,10 @@
 {
 	"info": {
-		"id": "pt-br.neoxscans",
+		"id": "pt-br.nexoscans",
```

Comment:
> Did not change id even if a source's name or language were changed

even thought the name changed, the id should stay the same that people can still update their old source.

Reply:
Done!